test/cell_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/cell'
+require './lib/ship'
+
+class ShipTest < Minitest::Test
+
+    def setup
+      cell = Cell.new("B4")
```

Comment:
I think these variables need to be changed to instance variables in order to run the test properly